src/pages/Login/LoginPage.tsx

```diff
@@ -72,6 +85,25 @@ export class LoginPage extends React.Component<LoginProps, LoginState> {
     if (authenticationConfig.strategy === AuthStrategy.openshift) {
       // If we are using OpenShift OAuth, take the user back to the OpenShift OAuth login
       window.location.href = authenticationConfig.authorizationEndpoint!;
+    } else if (authenticationConfig.strategy === AuthStrategy.token) {
```

Comment:
I think we are reaching a point here where either polymorphism or at least a `switch` case would be the better idea here...

Reply:
I agree. But given that this doesn't change a lot, let's wait. I'll be :+1: if one more auth method comes in.